mewarpx/mewarpx/logger.py

```diff
@@ -0,0 +1,29 @@
+"""Module for logging on the root proc"""
+
+from mewarpx.mwxrun import mwxrun
+
+import logging
+initialized = False
+
+class Logger(object):
+    def __init__(self):
+        self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.INFO)
+        self.logger.filters = [lambda _: mwxrun.me == 0]
+
+    def debug(self, message):
+        self.logger.debug(message)
+
+    def info(self, message):
+        self.logger.info(message)
+
+    def warning(self, message):
+        self.logger.warn(message)
+
+    def error(self, message):
+        self.logger.error(message)
+
+    def critical(self, message):
+        self.logger.critical(message)
+
+logger = Logger()
```

Comment:
This shouldn't be done with a special logger (use my original syntax for that), but with a special _handler_. See eg https://docs.python.org/3/howto/logging-cookbook.html#multiple-handlers-and-formatters . This will probably replace what's currently in `mewarpx/__init__.py`

Reply:
Also see https://docs.python.org/3/howto/logging.html#logging-flow